python/src/spark_rapids_ml/umap.py

```diff
@@ -538,19 +678,22 @@ def _construct_umap() -> CumlT:
 
             from .utils import cudf_to_cuml_array
 
-            embedding_np = np.array(self.embedding, dtype=np.float32)
-            raw_data_np = np.array(self.raw_data, dtype=np.float32)
+            embedding = self.embedding
+            raw_data = self.raw_data
```

Comment:
If changing the property type per above, would need to modify these accordingly.